src/app/dataset-view/dataset.subscriptions.interface.ts

```diff
@@ -38,6 +39,7 @@ export interface MetadataSchemaUpdate {
 export interface DatasetHistoryUpdate {
     history: MetadataBlockFragment[];
     pageInfo: DatasetPageInfoFragment;
+    kind?: DatasetKind;
```

Comment:
Why optionally undefined?

Reply:
In other part of app, we don't need this property.